composer.json

```diff
@@ -25,7 +25,7 @@
         "php":                      "^7.0",
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
-        "sebastian/exporter":       "^1.0 || ^2.0 || ^3.0",
+        "sebastian/exporter":       "^3.1",
```

Comment:
I would revert this change, as it makes it harder for projects to avoid a compatibility hell (this is a normal requirement, so it affects users). Version 2 of the exporter is also compatible with PHP 7.
As the API of the exporter is the same, there is absolutely no maintenance cost here.